build.gradle

```diff
@@ -81,6 +81,8 @@ dependencies {
     implementation 'info.picocli:picocli:4.0.4'
     implementation 'org.apache.commons:commons-csv:1.7'
 
+    implementation "io.grpc:grpc-services:1.30.0"
```

Comment:
I assume this should be the same grpc version used in the clientlib. Can we make the version a property in the root project and reference it in both places?